libs/schemas/src/collectibles.ts

```diff
@@ -15,6 +15,13 @@ export enum IPFSStatus {
   Stored = 'stored',
 }
 
+export enum CollectibleAuctionStatus {
```

Comment:
I think we might need an "Upcoming" status, should be added here, or would we compare now and startAt?